### PR TITLE
Auto-close menu after selecting an option using webflow interaction - Import latest webflow 2021 02 04

### PR DESCRIPTION
**Pull request by schalkneethling**
This might be good to have two sets of eyes on so, I am going to request a review from you both. I believe I have caught all the things that changed, but please double-check and let me know should I have missed anything. Thanks!

### CODE REVIEW

**Review by otterbotter — COMMENTED** [review]
I reviewed the diffs and they look good to me. I haven't worked with webflow importing/exporting yet so my input here is minimal. 

The only question I have is that the PR title/body indicates it is only updating the webflow export, but it looks to me like there are other unrelated changes to the navbar? Is it due to something changing with the export?

Specifically src/js/utils/menu.js

**Comment by schalkneethling** [comment]
> The only question I have is that the PR title/body indicates it is only updating the webflow export, but it looks to me like there are other unrelated changes to the navbar? Is it due to something changing with the export?

Indeed. So, there used to be some DOM elements "pre-populated" in the menu which have now been removed. As such, I had to update the way the menu JS adds, shows, and hide things. It is actually simpler now :) Thank you for taking a look @otterbotter

File: src/js/components/link-block.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export class LinkBlock {
   constructor(props) {
+    console.log(props);
```
Comment (jbothma) [inline src/js/components/link-block.js:3]: stray debug log

File: src/js/utils/menu.js [hunk 2]
```diff
@@ -29,42 +29,44 @@ export const updateMenuLinks = () => {
...
+  );
+  const $settingsLink = getAnchorElement(
+    "/accounts/settings/",
+    "nav-link w-inline-block",
+    "Settings"
+  );
 
-  let $logoutButton = $("#my-muni-logout");
+  $navMenu.find(".nav-link").remove();
+  $navMenu.append([$loginLink, $logoutButton, $registerLink, $settingsLink]);
```
Comment (jbothma) [inline src/js/utils/menu.js:55]: This should make auto-close on click work

```suggestion
  $navMenu.append([$loginLink, $logoutButton, $registerLink, $settingsLink]);
  Webflow.require('ix2').init();
```

**Pull request merged**